deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py

```diff
@@ -0,0 +1,339 @@
+import argparse
```

Comment:
The statistical analysis should be planned as a library instead of a script. This library should include a class or several classes that will handle calculating any statistical value or intermediate value regarding the provided data.

- **Modularity and Reusability**: By organizing the statistical analysis as a library, it becomes easier to reuse code across different projects. The classes and methods can be imported and utilized wherever needed without duplicating code.

- **Maintainability**: A well-structured library with clear class definitions and methods is easier to maintain and update. Bugs can be fixed, and new features can be added with minimal disruption to the existing codebase.

- **Scalability**: As data analysis needs to grow, the library can be expanded with additional classes and methods. This allows for the easy integration of new statistical techniques and algorithms without having to overhaul the entire codebase.

- **Encapsulation**: By using classes, data, and methods that operate on the data can be encapsulated together. This promotes better organization and prevents variables and methods from polluting the global namespace, reducing the risk of conflicts and errors.

- **Testing**: A library can be more easily tested with unit tests, ensuring each class and method performs as expected. This enhances the reliability of the code and helps catch issues early in the development process.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/5622/commits/26dc5cfa8d7f2339d13d5d5820d1ad7cceda3625